include/boost/beast/http/impl/verb.ipp

```diff
@@ -135,7 +120,7 @@ string_to_verb(string_view v)
         break;
 
     case 'B':
-        if(v == "IND")
+        if(v =="IND")
```

Comment:
How do you know this improves the performance? After the change it constructs a `string_view` from `char const*` which causes the string to be iterated twice? Doesn't it depend on which stdlib is being used (e.g. constexpr constructor)?

Reply:
Replaced it with an explicit conversion via a UDL to guarantee that `strlen` is not called.